src/GafferSceneUI/SetMembershipInspector.cpp

```diff
@@ -216,14 +216,33 @@ IECore::ConstObjectPtr SetMembershipInspector::value( const GafferScene::SceneAl
 	return matchResult & IECore::PathMatcher::Result::ExactMatch ? new BoolData( true ) : nullptr;
 }
 
-IECore::ConstObjectPtr SetMembershipInspector::fallbackValue( const GafferScene::SceneAlgo::History *history ) const
+IECore::ConstObjectPtr SetMembershipInspector::fallbackValue( const GafferScene::SceneAlgo::History *history, std::string &description ) const
 {
 	const auto &path = history->context->get<ScenePlug::ScenePath>( ScenePlug::scenePathContextName );
 	ConstPathMatcherDataPtr setMembers = history->scene->set( m_setName );
 
 	auto matchResult = (PathMatcher::Result)setMembers->readable().match( path );
 
-	return new BoolData( matchResult & IECore::PathMatcher::Result::AncestorMatch );
+	const bool ancestorMatch = matchResult & IECore::PathMatcher::Result::AncestorMatch;
+	if( ancestorMatch )
+	{
+		ScenePlug::ScenePath currentPath( history->context->get<ScenePlug::ScenePath>( ScenePlug::scenePathContextName ) );
```

Comment:
As for AttributeInspector, should we start from the parent here? I suppose the fact that `fallbackValue()` is being called does currently guarantee that there isn't an exact match at the current location, but it seems plausible that in the future we'd want to call `fallbackValue()` even when `value()` was non-null. For a sort of "what value will I end up with if I remove this value?" scenario.

Reply:
Sounds good to me, as with the AttributeInspector, this does seem more generally useful: 3d8823f.